lxd/patches.go

```diff
@@ -141,3 +145,816 @@ func patchNetworkPermissions(name string, d *Daemon) error {
 
 	return nil
 }
+
+func patchStorageApi(name string, d *Daemon) error {
+	lvmVgName := daemonConfig["storage.lvm_vg_name"].Get()
+	zfsPoolName := daemonConfig["storage.zfs_pool_name"].Get()
```

Comment:
I'm not seeing any code that clears those after migration, did I miss something or are we ending up with global config keys that are set and can't ever be cleared since they're now marked deprecated?